test/index.js

```diff
@@ -243,7 +248,12 @@ describe('hapi-setup Plugin', function () {
           },
           attributes: {
             name: 'bar',
-            version: '1.0.0'
+            version: '1.0.0',
+            pkg: {
+              version: '0.0.0'
```

Comment:
Actually I'm really surprised you needed to change this at all? Can you please explain why this change was necessary? The plugin interface hasn't change in like 5 versions of hapi.